package.json

```diff
@@ -368,11 +368,11 @@
     ]
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32256&commitId=a9bcfa968b41965c44faf7fc450abc0ea8b44579&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|3| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|16| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1146| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>